### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- bump dependencies
- https://github.com/kyma-project/eventing-manager/pull/509
- https://github.com/kyma-project/eventing-manager/pull/508
- https://github.com/kyma-project/eventing-manager/pull/507
- https://github.com/kyma-project/eventing-manager/pull/499

**Related issue(s)**

### CODE REVIEW

File: pkg/k8s/client_test.go [hunk 1]
```diff
@@ -26,17 +27,11 @@ import (
 
 const testFieldManager = "eventing-manager"
 
+var errPatchNotAllowed = errors.New("apply patches are not supported in the fake client")
+
 func Test_PatchApply(t *testing.T) {
```
Comment (muralov) [inline pkg/k8s/client_test.go:32]: I think, we should get rid of the test for PatchApply at all as we test it as a part of the integration tests. What do you think?

Comment (mfaizanse) [reply pkg/k8s/client_test.go:32]: I have no preference. We can keep it as having a test with simple checks is also a good idea.

Comment (muralov) [reply pkg/k8s/client_test.go:32]: nitpick: please delete it. Little bit less test code :)

Comment (mfaizanse) [reply pkg/k8s/client_test.go]: done

**Review by muralov — APPROVED** [review]

**Pull request merged**